Tlane77/Asclepius
# TL Initial Comment Files Client and Server Side

# Description
As a reader, I would like to see a list of all the Comments on a Condition so that I can read and take part in the discussion on a particular Condition.

Given the user is viewing the Details of a Condition
When they select the View Comments button
Then they should be directed to the Comments list page for the Condition
And the list should be in order of creation date with the most recent on top
And the title of the related Condition should be displayed at the top of the page
And a link back to the Conditions should be available

Display the following information for each Comment

Subject
Content
Author's Display Name
Creation date (MM/DD/YYYY)
Choose a Health Category on the Condition from a Drop Down Menu.
#### Styling
Added 10% margins to either side of window for all views to create a whitespace border.

# Ticket Reference

> [#7  - View/Add Categories](https://github.com/nss-day-cohort-41/Asclepius/compare/tlComments/issues/7)
> [#8  - View Conditions with Comments](https://github.com/nss-day-cohort-41Asclepius/compare/tlComments/issues/8)


## Type of change


# Testing Instructions
- Start program
- Go to localhost:3000 in your browser.
- Login with "tech3@health.com"
- Navigate to the "condition" link in the navbar.
-View the Conditions List
-Add a Condition
-Use Drop Down Menu to test Categories
-Add a Comment on the Condition
-Visually confirm the comments on the condition
- Next, click the logout button on the navbar.

# Checklist:
